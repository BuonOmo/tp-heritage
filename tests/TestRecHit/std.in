R rect 56 2 4 5
HIT rect 20 3
HIT rect 1 1
HIT rect 100 100
HIT rect 56 12
HIT rect 56 1
HIT rect 4 6
HIT rect 4 1
HIT rect 2 2
HIT rect 57 2
HIT rect 3 5
HIT rect 58 5
HIT rect 56 2
HIT rect 4 5
HIT rect 56 5
HIT rect 4 2
EXIT
