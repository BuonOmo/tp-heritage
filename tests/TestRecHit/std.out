OK
YES
NO
NO
NO
NO
NO
NO
NO
NO
NO
NO
YES
YES
YES
YES
